doc/latex.rst

```diff
@@ -1477,7 +1477,12 @@ Macros
      ``\sphinxkeyboard``
 
   .. versionadded:: 6.2.0
-     ``\sphinxparam``, ``\sphinxsamedocref``
+
```

Comment:
arrgh, this empty line is a left-over from re-flowing paragraph.  It does not matter for output, nevertheless I will push a commit to remove. Sorry.